cmd/eksctl-anywhere/cmd/createpackages.go

```diff
@@ -43,7 +43,7 @@ var createPackagesCommand = &cobra.Command{
 
 func createPackages(ctx context.Context) error {
 	kubeConfig := kubeconfig.FromEnvironment()
-	deps, err := newDependenciesForPackages(ctx, kubeConfig)
+	deps, err := curatedpackages.NewDependenciesForPackages(ctx, kubeConfig)
```

Comment:
If this method is using the factory, I would advice against putting it in the curatedpackages package

The dependency direction should be "ports" (like cmd or controllers) -> factory -> "packages with business logic"

If we break that, we will eventually find a cyclic dependency

Reply:
Addressed this offline: creating an issue to handle this separability